zipkin-lens/src/translations/es/messages.json

```
@@ -12,6 +12,7 @@
   "End Time": "Tiempo de Fin",
   "Failed to load this file": "Fallo al cargar este archivo",
   "Filter": "Filtro",
+  "Help": "Ayuda",
```

if changing to "Support": 

```suggestion
  "Help": "Soporte",
```